### PR TITLE
feat(api-client): Add getGiphyById endpoint

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: packages/api-client/src/giphy/GiphyAPI.ts [hunk 1]
```diff
@@ -32,6 +32,21 @@ export class GiphyAPI {
     SEARCH: 'search',
   };
 
+  /**
+   * Get a GIF by ID
+   * @param id ID of GIF image
```
Comment (ffflorian) [inline packages/api-client/src/giphy/GiphyAPI.ts]:
```suggestion
   * @param one or multiple image ID(s)
```

File: packages/api-client/src/giphy/GiphyAPI.ts [hunk 2]
```diff
@@ -32,6 +32,21 @@ export class GiphyAPI {
...
+   */
+  public async getGiphyById(ids: string | string[]): Promise<GiphyResult> {
+    ids = [].concat(ids as any);
```
Comment (ffflorian) [inline packages/api-client/src/giphy/GiphyAPI.ts]:
```suggestion
    const allIds = [].concat(ids);
```

File: packages/api-client/src/giphy/GiphyAPI.ts [hunk 3]
```diff
@@ -32,6 +32,21 @@ export class GiphyAPI {
...
+    const config: AxiosRequestConfig = {
+      method: 'get',
+      url: `${GiphyAPI.URL.PROXY}/${GiphyAPI.URL.GIPHY}/${ids.join(',')}`,
```
Comment (ffflorian) [inline packages/api-client/src/giphy/GiphyAPI.ts]:
```suggestion
      url: `${GiphyAPI.URL.PROXY}/${GiphyAPI.URL.GIPHY}/${allIds.join(',')}`,
```

File: packages/api-client/src/giphy/GiphyResult.ts [hunk 4]
```diff
@@ -30,4 +30,9 @@ export interface GiphyResult {
 
 export type GiphySearchResult = GiphyResult & {
   data: GiphyImage[];
+  pagination: {
+    total_count: number;
```
Comment (bennyn) [inline packages/api-client/src/giphy/GiphyResult.ts]: sorting

File: packages/api-client/src/giphy/GiphyAPI.ts [hunk 5]
```diff
@@ -32,6 +32,21 @@ export class GiphyAPI {
     SEARCH: 'search',
   };
 
+  /**
+   * Get a GIF by ID
+   * @param ids one or multiple image ID(s)
```
Comment (bennyn) [inline packages/api-client/src/giphy/GiphyAPI.ts]: Are these image IDs or Giphy IDs? Where do I get such ID?

File: packages/api-client/src/giphy/GiphyAPI.ts [hunk 6]
```diff
@@ -32,6 +32,21 @@ export class GiphyAPI {
     SEARCH: 'search',
   };
 
+  /**
+   * Get a GIF by ID
```
Comment (bennyn) [inline packages/api-client/src/giphy/GiphyAPI.ts]:
```suggestion
   * Get a Giphy image by its ID.
```

File: packages/api-client/src/giphy/GiphyAPI.ts (hunk 5 above)
Comment (Yserz) [reply packages/api-client/src/giphy/GiphyAPI.ts]: By nature of the Giphy API this is a giphy ID. You can get such an ID by searching for a gif with the other two endpoints.

**Pull request merged**